mimic/test/test_nova.py

```diff
@@ -39,6 +39,27 @@ def setUp(self):
         self.server_id = self.create_server_response_body['server']['id']
         self.nth_endpoint_public = helper.nth_endpoint_public
 
+    def test_create_server_with_manual_diskConfig(self):
+        """
+        Servers should respect the provided OS-DCF:diskConfig setting if
+        supplied.
+        """
+        create_server = request(
+            self, self.root, "POST", self.uri + '/servers',
+            json.dumps({
+                "server": {
+                    "name": self.server_name + "A",
+                    "imageRef": "test-image",
+                    "flavorRef": "test-flavor",
+                    "OS-DCF:diskConfig": "MANUAL",
+                }
+            }))
+        create_server_response = self.successResultOf(create_server)
+        response_body = self.successResultOf(
+            treq.json_content(create_server_response))
+        self.assertEqual(
+            response_body['server']['OS-DCF:diskConfig'], 'MANUAL')
```

Comment:
Sure @sam-falvo.

I'm just talking about the thoroughness of the test, not the correctness of the implementation.

This test is asserting only that the HTTP response to the creation `POST` looks a certain way; it doesn't assert anything about subsequent requests.  So an implementation which simply passed the input through to the output but *didn't* store the disk configuration anywhere internally would still pass the test.  There doesn't seem to be any other test verifying this either, so an implementation change which breaks in this way wouldn't be caught.